src/Symmetric/Openssl.php

```diff
@@ -167,6 +167,19 @@ class Openssl implements SymmetricInterface
      */
     protected $tagSize = 16;
 
+    /**
+     * Supported algo list
+     *
+     * This property was previously dynamic. It should be made private in the next major, therefore marked internal.
+     *
+     * @internal
+     *
+     * @link self::getSupportedAlgorithms()
+     *
+     * @var list<string>|null
```

Comment:
`|null` conflicts with the typed property declaration

Reply:
👍 removed the type for now. There's no psalm in this repo either.